no-computer.md

```diff
@@ -7,45 +7,103 @@ good way to meet their needs.
 
 
 ### Expressiveness
-_What should be easy to do in this language? What should be possible, but
-difficult? What should be impossible or very difficult?_
+_What should be easy to do in this language?_
+
+Braille not only serves as a replacement for printed texts in many languages, but can also use the concept of raised dots to communicate an image. Given Braille’s origins as a language derived from the Latin alphabet, and given the fact that its letters were organized according to their position within the French alphabet, it should also be easy to represent other phonetic languages in Braille.
+
+_What should be possible, but difficult?_
+
+However, it might be difficult to represent non-phonetic languages in Braille (excluding non-phonetic languages that have romanized representations, such as Chinese), since many non-phonetic languages do not require the reader to read all the components of a word / character in succession from left to right or right to left. 
+
+_What should be impossible or very difficult?_
+
+Text can often be presented in different fonts, but a font would be difficult to capture in Braille. The challenge is that the variations between different fonts would still need to ensure that the letter is recognizable to the touch. 
+Another challenge that Braille faces might be expanding the language. Even though English Braille alphabet is composed of all letters and symbols, what if there is a new symbol that needs to be represented, but the alphabet has already run out of possible combinations of 6 raised/non raised dots? What if Braille also wanted to represent symbols such as emojis? 
 
 
 ### Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
-aren't and conjecture why not. If so, describe them and provide links. How well 
-do they address the need? Are there any particularly admirable qualities of the
-language? Are there parts of the language you think could be improved?_
+aren't and conjecture why not. If so, describe them and provide links._
+
+Another language that the blind/visually impaired could use to communicate and understand thought is through sign language that is spelled out into the hands of someone who is blind or visually impaired. 
 
+http://www.lifeprint.com/ This website provides information and resources to help people learn ASL and improve their signing. 
+
+_How well do they address the need?_
+
+I think Braille is preferred to sign language because in order for a visually impaired person to receive and communicate information via sign language, he or she would need another person there to perform the sign language or speak it. It would be really interesting to hear of software that receives sign language footage as input and produces a sheet of paper with Braille.
+
```

Comment:
I agree, signing into the palm of someone's hand seems like extra work.  If a visually-impaired person could hear, and another person were present to sign into their palm, it seems easier to merely say the words aloud.